src/main/java/hudson/plugins/s3/callable/S3GzipCallable.java

```diff
@@ -46,21 +45,28 @@ private File gzipFile(FilePath file) throws IOException, InterruptedException {
     }
 
     // Hook to ensure that the file is deleted once the upload finishes.
-    private static class CleanupHook implements ProgressListener {
+    private static class CleanupHook implements TransferListener {
         private final File localFile;
 
         CleanupHook(File localFile) {
             this.localFile = localFile;
         }
 
         @Override
-        @SuppressFBWarnings({ "RV_RETURN_VALUE_IGNORED_BAD_PRACTICE", "SF_SWITCH_NO_DEFAULT" })
```

Comment:
This was fun; if I removed the suppression, then spotbugs complains about the return value of `localFile.delete();` being ignored, but if I add it, then spotbugs complains that the suppression is unnecessary :exploding_head: